pkg/descheduler/evictions/evictions.go

```diff
@@ -225,6 +239,15 @@ func (pe *PodEvictor) Evictable(opts ...func(opts *Options)) *evictable {
 			return nil
 		})
 	}
+	//if options.nodeFit != nil && *options.nodeFit {
+	if options.nodeFit {
+		ev.constraints = append(ev.constraints, func(pod *v1.Pod) error {
+			if !nodeutil.PodFitsAnyOtherNode(pod, pe.nodes) {
+				return fmt.Errorf("pod does not fit on any other node because of nodeSelector(s), Taint(s), or nodes marked as unschedulable")
```

Comment:
nodeSelector(s), Taint(s), -> node selector, node affinity, taint(s) mismatch,

Reply:
Got it, thank you.